create-hpc-cluster-linux-cn/azuredeploy.json

```diff
@@ -159,400 +159,122 @@
     }
   },
   "variables": {
-    "apiVersion": "2015-05-01-preview",
-    "location": "[resourceGroup().location]",
     "virtualNetworkName": "[concat(parameters('clusterName'), 'VNet')]",
     "storageAccountNamePrefix": "[toLower(concat('hpc', uniqueString(resourceGroup().id, parameters('clusterName'))))]",
```

Comment:
@singhkay Thanks for the advice for storage account naming. But the name here is for storageAccountNamePrefix, so whatever the prefix is, we will create a series of storage accounts with same prefix. Putting the uniqueString at the beginning of the nameprefix doesn't really help to distribute the storage accounts on different racks.